JapaneseFilter/sections/specific.txt

```diff
@@ -1341,6 +1341,8 @@ bipblog.com###TopAmazon
 bipblog.com###BottomAmazon
 bipblog.com##.main_center_block[style^="margin"]
 bipblog.com##.main_left_block:nth-last-of-type(-n+4)
+bipblog.com##.amachazl-box
+bipblog.com###ent > iframe[src^="http://bipblog.com/"]
```

Comment:
Same as below, but `||bipblog.com/bipblog.html` can be added to Annoyances > widget as this leads to third-party sites.

Reply:
ok,done